docs/specification.md

```diff
@@ -411,6 +414,24 @@ Notification:
 export interface InitializedBuildParams {}
 ```
 
+### Reload request
+
+The `reload` request instructs the build server to reload the build configuration.
```

Comment:
Normally, the request/notification descriptions explicitly say whether its sent from the client or server.

```suggestion
The `reload` request is sent from the client to instruct the build server to reload the build configuration.
```

Reply:
added it!